src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -74,3 +74,7 @@ GENERATE_EXPORT_HEADER( clrcompression
 )
 
 install (TARGETS clrcompression DESTINATION .)
+
+install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/clrcompression.pdb DESTINATION .
+    CONFIGURATIONS Debug
```

Comment:
Thanks, I read up a bit on it and yeah, that being a blocker makes sense. I'll see if I can get the build creating and packaging the release PDB.